gemrb/plugins/BIKPlayer/common.h

```diff
@@ -33,7 +33,11 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
- 
+
+#ifdef __amigaos4__
+#include <malloc.h> //memalign
```

Comment:
this should be included by stdlib. can you try changing `#include <stdlib.h>` to `#include <cstdlib>` and see if that fixes it?

Reply:
actually, our code doesn't _require_ `memalign`, what is this really about?

Reply:
no, unlucky our newlib define memalign in malloc.h